From f6c1f9e5924246216a05c295f450989ad9c80104 Mon Sep 17 00:00:00 2001
From: Maksym Sobolyev <sobomax@sippysoft.com>
Date: Thu, 13 May 2021 11:42:29 -0700
Subject: [PATCH] Resolve build breakage after ekt member of the policy struct
 has been deprecated by the libsrtp. We have not been using that member to do
 anything meaningful anyhow and the struct is inied to \0, so it should have
 no adverse effect.

---
 extractaudio/eaud_crypto.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/extractaudio/eaud_crypto.c
+++ b/extractaudio/eaud_crypto.c
@@ -153,7 +153,7 @@ eaud_crypto_getopt_parse(char *optarg)
     srtp_crypto_policy_set_rtp_default(&rval->policy.rtp);
     srtp_crypto_policy_set_rtcp_default(&rval->policy.rtcp);
     rval->policy.key = (uint8_t *)rval->key;
-    rval->policy.ekt = NULL; rval->policy.next = NULL;
+    rval->policy.next = NULL;
     rval->policy.window_size = 128;
     rval->policy.allow_repeat_tx = 0;
     rval->policy.rtp.auth_tag_len = suite->tag_size;
